assets/publish_lambda_layer.sh

```diff
@@ -1,33 +1,34 @@
-export CID_VERSION=`python3 -c "from cid import _version;print(_version.__version__)"`
+export CID_VERSION=$(python3 -c "from cid import _version;print(_version.__version__)")
 
-mkdir -p ./python
-pip3 install . -t ./python
-zip -r cid-$CID_VERSION.zip ./python
+tmpdir=$(mktemp -u)
+mkdir $tmpdir
+pip3 install . -t $tmpdir
+zip -r cid-$CID_VERSION.zip $tmpdir
 ls cid-$CID_VERSION.zip
-rm -rf ./python
-
+rm -rf $tmpdir
 
+export AWS_REGION=us-east-1
```

Comment:
Can we keep REGION loop to reuse the same script in the prod update?

Reply:
there is a loop, this one is needed for local run